docs/software/advanced_networking.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Advanced Networking Software Packages
```

Comment:
Since the title by default matches the navigation menu entry, for software categories it is "Advanced Networking - DietPi.com Docs" here OOTB. Do you think it is valuable to have the "Software Packages" (I would the use "Software Options") appended instead?

Reply:
Packages seemed a bit more user-friendly for me. But options is also fine for me.
Changed

Reply:
> For me, options is more a phrase of choices for the user he needs to select.

I understand. "packages" for me sounds not so fitting since many (at least me) use this more for things like APT (package manager) packages, when one can talk about everything being "packed" into a single file to install, which does not apply to how `dietpi-software` works. Quite a good question which word describes best optional software installs we offer 🤔.

Skipping the title and hence falling back to navigation menu text + " - DietPi.com Docs" would avoid this question 😄.

Reply:
The appendix is still added. Okay, let's leave your suggestions in place here, a category only indeed looks a bit strange.